web-common/src/components/column-profile/column-types/sparks/TimestampSpark.svelte

```diff
@@ -14,59 +14,17 @@
     Line,
   } from "@rilldata/web-common/components/data-graphic/marks";
 
-  //export let data;
-
   export let width = undefined;
   export let height = undefined;
   export let data: unknown[];
-  // export let curve = "curveLinear";
-  // export let area = false;
-  // export let color = "hsl(217, 10%, 50%)";
-  // export let tweenIn = false;
-
-  // // the color of the zoom window
-  // export let zoomWindowColor = "hsla(217, 90%, 60%, .2)";
-  // // the color of the zoom window boundaries
-  // export let zoomWindowBoundaryColor = "rgb(100,100,100)";
-  // export let zoomWindowXMin: Date = undefined;
-  // export let zoomWindowXMax: Date = undefined;
 
   export let xAccessor: string | undefined = undefined;
   export let yAccessor: string | undefined = undefined;
 
-  // rowsize for table
-
   export let left = 0;
   export let right = 0;
   export let top = 12;
   export let bottom = 4;
-
-  // export let buffer = 4;
-  // export let leftBuffer = buffer;
-  // export let rightBuffer = buffer;
-  // export let topBuffer = buffer;
-  // export let bottomBuffer = buffer;
-
-  // export let objectName: string;
-  // export let columnName: string;
-
-  // $: sparkQuery = createQueryServiceColumnTimeSeries(
-  //   $runtime?.instanceId,
-  //   // FIXME: convert pixel back to number once the API
-  //   {
-  //     tableName: objectName,
-  //     timestampColumnName: columnName,
-  //     pixels: 92,
-  //   }
-  // );
-  // let data = [];
-  // $: data = convertTimestampPreview(
-  //   $sparkQuery?.data?.rollup?.spark?.map((di) => {
-  //     let next = { ...di };
-  //     next[yAccessor] = next.records[yAccessor];
-  //     return next;
-  //   }) || []
-  // );
```

Comment:
Note: changes in this file are just opportunistic cleanup that I found while searching "color" this code was commented out 12-14 months ago.